### ISSUE TITLE
trains need text field that supersedes cost

### ISSUE THREAD

**Issue by th3tick** [18xx-maker/18xx-maker#305]
This is useful for privates which come with a train.  Replacing the price with text makes it more obvious you can't just buy it.

**Comment by kelsin** [comment]
Yeah we should do this on almost any place with a numeric cost (which I want to standardize). Basically for every field named `x` that's a numeric cost or value, we should have a `xFormat` that is a format string that replaces `#` in the string with the value.
